common/types/commitments/mode.go

```diff
@@ -14,6 +14,17 @@ const (
 	StandardCommitmentMode        CommitmentMode = "standard"
 )
 
+// EncodeCommitment serializes the versionedCert prepends commitmentMode-related header bytes.
```

Comment:
nit: 

"serializes the versionedCert *and* prepends ..."

Reply:
I wish I could make this change but then it would dismiss your approval and force me to wait another 8 hours before merging. I'm really starting to feel like with the timezone differences we should remove that check. Much easier for me to wake up, make a few nit commits to address the comments, and then merge. If there are large changes required that necessitate another round of review I would of course wait before merging. WDYT?

Reply:
@samlaf Unfortunately the only good solution to this problem is to have at least one additional reviewer in a similar timezone :/

I think it's ok if we remove this protection. The hassle outweighs the benefit, given the current makeup of our team.